src/creative.js

```diff
@@ -29,18 +29,19 @@ const DEFAULT_CACHE_PATH = '/pbc/v1/cache';
 /**
  * Public render ad function to be used in dfp creative setup
  * @param  {object} doc
- * @param  {string} adId
  * @param  {dataObject} dataObject
  */
-pbjs.renderAd = function(doc, adId, dataObject) {
-  if(environment.isMobileApp(dataObject)) {
-    renderAmpOrMobileAd(dataObject.cacheHost, dataObject.cachePath, dataObject.uuid, dataObject.size, true);
-  } else if (environment.isAmp(dataObject)) {
-    renderAmpOrMobileAd(dataObject.cacheHost, dataObject.cachePath, dataObject.uuid, dataObject.size);
+ucTag.renderAd = function(doc, dataObject) {
+  const auctionData = utils.transformAuctionTargetingData(dataObject);
```

Comment:
Can you rename auctionData. It store targeting key values so maybe something like `targetingData` `targetingInfo` etc